src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -398,7 +398,7 @@ namespace Microsoft.FSharp.Core
         let inline unboxPrim<'T>(x:obj) = (# "unbox.any !0" type ('T) x : 'T #)
         let inline box     (x:'T) = (# "box !0" type ('T) x : obj #)
         let inline convPrim<'T1, 'T2>(x: 'T1) : 'T2 = unboxPrim<'T2> (box x) 
-        let inline not     (b:bool) = (# "ceq" b false : bool #)
+        let inline not     (b:bool) = if b then false else true
```

Comment:
This change is probably the source of my woes w.r.t IL baselines, I wonder why though